pkgs/top-level/all-packages.nix

```diff
@@ -7650,7 +7650,7 @@ with pkgs;
 
   dvtm-unstable = callPackage ../tools/misc/dvtm/unstable.nix { };
 
-  a4term = callPackage ../tools/misc/a4term { };
+  a4 = callPackage ../tools/misc/a4 { };
```

Comment:
If you will rename this, then move the file to `pkgs/by-name`.

And, obviously, do not forget to throw the old name.